app/controllers/features_controller.rb

```diff
@@ -1,12 +1,45 @@
 class FeaturesController < ApplicationController
   def create
-    @user = User.find(params[:plan_id])
-    @plan = @user.plans.create(feature_params)
-    redirect_to welcome_index_path
+    @feature = Feature.create(feature_params)
+    redirect_to features_path
+  end
+
+  def index
+    @feature = Feature.all
```

Comment:
variable names should be describing the type it holds. In this case it should be features.

Reply:
Corrected.